src/test/java/teammates/test/driver/PriorityInterceptor.java

```diff
@@ -34,86 +34,82 @@
     // 4) Orders methods by class name in lexical order
     // 5) Orders methods by priority e.g. Add "@Priority(1)" to method
 
-    @Override
     @SuppressWarnings("deprecation")
+    private int getMethodPriority(IMethodInstance mi) {
+        int result = 0;
+        Method method = mi.getMethod().getMethod();
+        Priority a1 = method.getAnnotation(Priority.class);
+        if (a1 != null) {
+            result = a1.value();
+        }
+        return result;
+    }
+
+    @SuppressWarnings("deprecation")
+    private int getClassPriority(IMethodInstance mi) {
+        int result = 0;
+        Method method = mi.getMethod().getMethod();
+        Class<?> cls = method.getDeclaringClass();
+        Priority classPriority = cls.getAnnotation(Priority.class);
+        if (classPriority != null) {
+            result = classPriority.value();
+        }
+        return result;
+    }
+
+    @SuppressWarnings("deprecation")
+    private String getPackageName(IMethodInstance mi) {
+        return mi.getMethod().getMethod().getDeclaringClass().getPackage().getName();
+    }
+
+    @SuppressWarnings("deprecation")
```

Comment:
Since we're touching all of these lines, see if you can find the cause of all the above deprecations and fix them accordingly.

Reply:
The `getMethod()` function seems to be deprecated.